### ISSUE TITLE
KeyError when COLR layer doesn't exist in glyph for global layer mapping

### ISSUE THREAD

**Issue by justvanrossum** [justvanrossum/fontgoggles#143]
If there's a global COLR layer mapping, missing layers show a KeyError. This should probably be ignored silently, but compare with ufo2ft behavior. I'm fairly sure the intention is that color layers are optional, even for a font-global mapping.

**Comment by anthrotype** [comment]
ufo2ft just pass them on to fontTools.colorLib.builder.buildCOLR without doing anything besides parsing the plist.
What's a "missing" layer?

**Comment by anthrotype** [comment]
oh maybe you meant the preprocessor filter that creates the final layers, forgot about that (nanoemoji builds them beforehand)

**Comment by justvanrossum** [comment]
Yeah, it's about the mapping from `font.lib["com.github.googlei18n.ufo2ft.colorLayerMapping"]`.
